sources/packages/web/src/components/common/OfferingFormSubmit.vue

```diff
@@ -114,11 +112,15 @@ export default defineComponent({
     const validateOfferingData = async (
       data: EducationProgramOfferingAPIInDTO,
     ): Promise<OfferingValidationResultAPIOutDTO> => {
+      const typedData = excludeExtraneousValues(
+        EducationProgramOfferingAPIInDTO,
+        data,
+      );
       const validationResult =
         await EducationProgramOfferingService.shared.validateOffering(
           props.locationId,
           props.programId,
-          data,
+          typedData,
```

Comment:
In my local, Iam getting error for typedData
![image](https://github.com/bcgov/SIMS/assets/77353155/1b79c309-3f44-4d1c-84ad-379287eedf8a)

Reply:
never mind, it was throwing the error, because, I didn't install `class-transformer` on the web. resolving